lib/MetaCPAN/Document/Release.pm

```diff
@@ -591,5 +592,77 @@ sub requires {
     };
 }
 
+sub _activity_filters {
+    my ( $self, $params, $start ) = @_;
+    my ( $author, $distribution, $module, $f )
```

Comment:
Could we clarify what `$f` is?  It's not clear to me from the name or the context of the code.  :)